### PR TITLE
comments on review

### PR DESCRIPTION
**Pull request by HassanAlhaji**

### CODE REVIEW

File: src/components/CommentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    getCommets(reviewId).then((comments) => {
+      setComments(comments);
+    });
+  }, [reviewId]);
+
+  return <div>{comments.map((comment) => {
+    return <div className="comments" key= {comment.comment_id}>
+        <p>Author {comment.author}</p>
+        <p>{comment.body}</p>
+        <p>{comment.created_at}</p>
```
Comment (roisinoreilly) [inline src/components/CommentCard.jsx]: this timestamp isn't user friendly, try manipulating the data to display it or trying a package that will convert it into a readable date for you, I like moment.js

File: src/components/ReviewCard.jsx [hunk 2]
```diff
@@ -4,10 +4,17 @@ function ReviewCard({ review }) {
...
+        <Link to={`/review/${review.review_id}`}>{review.title}</Link>
+      </h3>
+      <img
+        src={review.review_img_url}
+        className="review-image"
+        alt="review pic"
+      ></img>
       <p>Category : {review.category}</p>
       <p> Designer : {review.designer}</p>
+      <button></button>
```
Comment (roisinoreilly) [inline src/components/ReviewCard.jsx]: this button isn't doing anything

**Review by roisinoreilly — APPROVED** [review]
looking great so far! I've left a couple of comments but merge when ready

File: src/components/CommentCard.jsx (hunk 1 above)
Comment (HassanAlhaji) [reply src/components/CommentCard.jsx]: Done.

File: src/components/ReviewCard.jsx (hunk 2 above)
Comment (HassanAlhaji) [reply src/components/ReviewCard.jsx]: Good spot, removed.

**Pull request merged**